app/main/views/frameworks.py

```diff
@@ -273,6 +273,89 @@ def framework_start_supplier_declaration(framework_slug):
                            framework_close_date=framework_close_date), 200
 
 
+@main.route('/frameworks/<framework_slug>/declaration/fdo', methods=['GET'])
+@login_required
+def fdo_placeholder(framework_slug):
+    # TODO remove this method entirely and its template.
+    reuse = data_api_client.get_supplier_framework_info(
+        current_user.supplier_id, framework_slug
+    )['prefillDeclarationFromFrameworkSlug']
```

Comment:
This dereference breaks - the API response looks like:
```
{u'frameworkInterest': {u'agreementId': None, u'agreementReturnedAt': None, u'agreementPath': None, u'supplierId': 700008, u'prefillDeclarationFromFrameworkSlug': u'digital-outcomes-and-specialists-2', u'countersignedPath': None, u'frameworkSlug': u'g-cloud-9', u'countersignedDetails': None, u'countersignedAt': None, u'agreementStatus': None, u'onFramework': None, u'agreementReturned': False, u'agreedVariations': {}, u'declaration': {}, u'countersigned': False, u'supplierName': u"Kev's Pies", u'agreementDetails': None}}
```
So it would need to be ['frameworkInterest']['prefillDeclarationFromFrameworkSlug']